src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -361,4 +282,128 @@ const Occurrences: React.FC<Props> = ({
   );
 };
 
+interface OccurrenceInfoProps {
+  occurrence: OccurrenceFieldsFragment;
+  event: EventFieldsFragment;
+  eventLocationId: string;
+}
+
+const OccurrenceInfo: React.FC<OccurrenceInfoProps> = ({
+  occurrence,
+  event,
+  eventLocationId,
+}) => {
+  const { placeId, startTime, endTime, linkedEvent } = occurrence;
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const date = formatDate(new Date(startTime));
+  const time =
+    startTime &&
+    formatTimeRange(new Date(startTime), new Date(endTime), locale);
+  const offer = linkedEvent?.offers?.[0];
+  const price = offer?.price?.[locale];
+  const priceDescription = offer?.description?.[locale];
+  const isFree = offer?.isFree ?? !price;
+  const priceInfoUrl = offer?.infoUrl?.[locale];
+
+  const createLink = (prefix: string, url: string) => (
+    <>
+      {prefix}
+      {prefix && ' '}
+      <a href={url} rel="noopener noreferrer" target="_blank">
+        {url}
+      </a>
+    </>
+  );
+
+  const renderOccurrenceTimeInfoSection = (
+    <>
+      <div>
+        <IconClock />
+      </div>
+      <div>
+        <div className={styles.infoTitle}>
+          {t('occurrence:dateAndTimeTitle')}
+        </div>
+        <div>{t('occurrence:textDateAndTime', { date, time })}</div>
+        <OccurrenceGroupInfo occurrence={occurrence} />
+      </div>
+    </>
+  );
+
+  const renderOccurrencePriceInfoSection = (
+    <>
+      <div>
+        <IconGlyphEuro />
+      </div>
+      <div>
+        <div className={styles.infoTitle}>
+          <div data-testid="event-price">
+            {isFree && t('event:occurrenceList.eventIsFree')}
+            {!isFree && price}
+          </div>
+        </div>
+        {!!priceDescription && (
+          <p data-testid="event-priceDescription">{priceDescription}</p>
+        )}
+        {!!priceInfoUrl && (
+          <p data-testid="event-priceInfoUrl">{createLink('', priceInfoUrl)}</p>
+        )}
+      </div>
+    </>
+  );
+
+  const renderOccurrenceLocationInfoSection = (
+    <>
+      <div>
+        <IconLocation />
+      </div>
+      <div>
+        <div className={styles.infoTitle}>{t('event:location.title')}</div>
+        <PlaceInfo
+          id={placeId || eventLocationId}
+          language={locale}
+          showVenueInfo
+          showPlaceInfoLinks
+        />
+      </div>
+    </>
+  );
+
+  const renderOccurrenceActions = (
+    <>
+      {/* TODO: functionality for these buttons */}
+      <CalendarButton event={event} occurrence={occurrence} />
+      <Button iconLeft={<IconLocation />} variant="supplementary">
+        {t('event:occurrenceList.showAllLocationEvents')}
+      </Button>
+      {/* Move map links down a bit show they would be 
+      closer to location info shown in left column  */}
+      <div style={{ height: '50px' }}></div>
```

Comment:
Might make more sense to wrap PlaceInfoLinks with container div and give that margin-top. But can't say there anything wrong with this, not a must change

Reply:
I was thinking about it too, but I didn't want to make a new (sub) container for half of the buttons. I thought this would be a cleaner solution, since it does not make the buttons different when comparing them to each other.

Reply:
OK. This approach is ok by me but now the div can be self closed :) `<div style={{ height: '50px' }} />`